### ISSUE TITLE
Add perl-MCE-Shared package

### ISSUE THREAD

**Issue by marioroy** [clearlinux/distribution#2688]
Official package name: perl-MCE-Shared

See also: https://src.fedoraproject.org/rpms/perl-MCE-Shared/tree/f36 and https://metacpan.org/pod/MCE::Shared.
```
perl Makefile.PL
make
```

License (must be an OSI approved Open Source license): GPL+ or Artistic

Download URL of latest release: https://cpan.metacpan.org/authors/id/M/MA/MARIOROY/MCE-Shared-1.877.tar.gz

Latest release date (must be recent): May 24, 2022

Description:

This module provides data sharing capabilities for MCE supporting threads and processes. The included MCE::Hobo module provides threads-like parallelization for running code asynchronously.

MCE::Shared requires MCE, Time::HiRes (included with Perl), Sereal::Encoder, Sereal::Decoder, and IO::FDPass.

Package request for MCE: https://github.com/clearlinux/distribution/issues/2687
Package request for IO::FDPass: https://github.com/clearlinux/distribution/issues/2686

Will perl-MCE-Shared be added to the perl-extras bundle? Thank you.

**Comment by marioroy** [comment]
Thank you! Output on Clear Linux (36710):
```bash
sudo swupd bundle-add perl-extras

perl -MMCE::Shared -le 'print $MCE::Shared::VERSION'
1.877
```

**Comment by fenrus75** [comment]
I;m adding these now

fwiw as long as things are in CPAN it's fine to just file 1 request for a
set of related packages;
we have automation in place to package CPAN content

**Comment by marioroy** [comment]
Oh! I didn't file a request recently at Clear Linux. What happened is that I released MCE 1.880 and MCE::Shared 1.878 yesterday on CPAN (Oct 10). That may have triggered something.

I met to send a message at the time of the initial request.
```text
perl-IO-FDPass has one dependency perl-common-sense
perl-MCE-Shared has two dependencies perl-MCE and perl-IO-FDPass
```

Some brief history on why MCE and MCE::Shared are separated. I once had one CPAN module MCE. But was asked by the Perl folks to separate MCE::Shared out as it requires minimally Perl 5.10.1. They wanted for MCE to support Perl 5.8.1 and higher.
